packages/docs/content/migration-guides/standardize-theme-colors.mdx

```diff
@@ -322,6 +322,105 @@ Red is removed and was a non-semantic duplicate of `error`.
   ]}
 />
 
+<ThemeContent onlyThemes={['medicare']}>
+
+<ColorMigrationTable
+  colorRows={[
+    [{ name: '--red-100',          value: '#f7e6e6' }, null],
+    [{ name: '--red-300',     value: '#e8b3b3' }, null],
+    [{ name: '--red-500',   value: '#b20000' }, null],
+    [{ name: '--red-700',  value: '#a00000' }, null],
+  ]}
+/>
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+### Teal
```

Comment:
@pwolfert and @forrestbaer not sure of the best way to format this Mgov specific content right now these headings like `Teal` show up on on table of contents for every theme though only applicable to Mgov :-/ 

Idea?

Reply:
I don't have a fix for it, but a workaround could be to drop them under another heading so all the individual colors don't show up. I'm trying to think of where to put things to achieve that.

I saw you put red under red, so in that same line of thinking,
- Teal could go under primary
- Blue could go under secondary
- Yellow could go under gold
- Black could go under neutral

Reply:
Thanks for the suggestion @pwolfert this is ready for 👀